tools/agat/agat.xml

```diff
@@ -136,12 +136,19 @@
                 #end if
                 -pe $tool.prot_existence
                 -o galaxy_functional_annot
+        #else if $tool.selector == 'keep_longest_isoform'
+            @input_annotation_single@
+            agat_sp_keep_longest_isoform.pl
+                --gff '$input_annotation'
+                --config '$agat_configfile'
+                --output ./output &&
+            cat ./output > '${annotation}'
```

Comment:
```suggestion
                --output '${annotation}'
```

We can do that in the other places as well. This extra call of cat is not needed.

Reply:
ok, I can change these